website/_layout.jade

```diff
@@ -12,4 +12,4 @@ html
     main
       != yield
 
-    script(src="js/application.js")
+    // script(src="js/application.js")
```

Comment:
This could probably be completely removed unless you're using any js. This is leftover from when i was going to script a terminal output. Should be ok to remove.